frontend/tests/README.md

```diff
@@ -0,0 +1,121 @@
+# Tests documentation
+
+## How to run E2E tests locally
+
+### What
+
+Running E2E tests locally means the tests are run in your local machine against the web app that's either running locally as well (localhost, docker) OR a deployed environment (dev, staging, prod)
```

Comment:
deployed environment including rdev?

Reply:
Ooh yes! Let me add a section for testing against an rdev 👌 Thank you!

Reply:
Done! 
<img width="869" alt="Screen Shot 2022-10-19 at 7 16 32 PM" src="https://user-images.githubusercontent.com/6309723/196840448-e3da8ade-281c-4e37-b5e2-103b801d9abb.png">